src/Esfa.Vacancy.Api.Types/CreateApprenticeshipParameters.cs

```diff
@@ -44,6 +44,11 @@ public class CreateApprenticeshipParameters
         /// </summary>
         public WageType WageType { get; set; }
 
+        /// <summary>
+        /// The reason for choosing the wage type
```

Comment:
Maybe this should be `The reason for choosing the 'Competitive Salary', 'To be confirmed' etc Wage Type`